packages/terra-notification-dialog/src/NotificationIcon.module.scss

```
@@ -0,0 +1,50 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './clinical-lowlight-theme/NotificationIcon.module';
+@import './orion-fusion-theme/NotificationIcon.module';
+
+:local {
+  .icon-container {
+    flex-shrink: 0; // This will keep text-wrapper from overlapping icon-container in IE11
+    height: var(--terra-notification-dialog-icon-container-height, 3rem);
+    margin: var(--terra-notification-dialog-icon-container-margin, 0);
```

Add 2px margin top to help icon align with titles
```suggestion
    margin: var(--terra-notification-dialog-icon-container-margin, 0.14285rem 0 0 0);
```

---

updated: https://github.com/cerner/terra-framework/pull/1105/commits/b06b04c4f64702b00a0f5e2e4159aadcd23bd23c#diff-900f722416924aaca1921d01d2a52e97R11